src/chrome/content/rules/Go_Walker.org.xml

```diff
@@ -1,4 +1,8 @@
+
 <!--
+Disabled by https-everywhere-checker because:
+Fetch error: http://www.gowalker.org/ => https://gowalker.org/: (51, "SSL: no alternative certificate subject name matches target host name 'www.gowalker.org'")
+
```

Comment:
Similar to the above example, this is actually because http://www.gowalker.org (the insecure version) redirects to https://www.gowalker.org/, which has an invalid certificate.